app/components/post_draft/quick_actions/quick_actions.tsx

```diff
@@ -48,30 +51,29 @@ export default function QuickActions({
     postPriority,
     updatePostPriority,
     focus,
+    handleToggleEmojiPicker,
+    isEmojiPickerFocused,
 }: Props) {
     const atDisabled = value[value.length - 1] === '@';
     const slashDisabled = value.length > 0;
 
     const atInputActionTestID = `${testID}.at_input_action`;
     const slashInputActionTestID = `${testID}.slash_input_action`;
-    const fileActionTestID = `${testID}.file_action`;
-    const imageActionTestID = `${testID}.image_action`;
-    const cameraActionTestID = `${testID}.camera_action`;
```

Comment:
This is probably breaking some e2e tests. Verify with @yasserfaraazkhan and get them fixed, or disabled them so we can fix them later.

Reply:
@yasserfaraazkhan Can you please help regarding this?